src/types.rs

```diff
@@ -199,3 +200,97 @@ pub enum OffsetEncoding {
     #[serde(rename = "utf-16")]
     Utf16,
 }
+
+#[derive(Debug, Eq, PartialEq, Clone, Default, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ServerCapabilities {
```

Comment:
Is it possible to use https://docs.rs/lsp-types/0.67.1/lsp_types/struct.ServerCapabilities.html directly?

Reply:
Oh Yeah, my mistake, we do use the lsp-types one. This was before I realized it would be better to just contribute to that project